### PR TITLE
CONN-1279: PLEASE DO NOT MERGE.

### PR DESCRIPTION
**Pull request by vsaibabu**

### CODE REVIEW

File: Product/Production/Gateway/pom.xml [hunk 1]
```diff
@@ -20,7 +21,8 @@
         <module>PatientDiscovery_10</module>
         <module>CONNECTGatewayWeb</module>
         <module>Direct</module>
-    </modules>
+		<module>X12DocumentSubmission_10</module>
```
Comment (christophermay07) [inline Product/Production/Gateway/pom.xml]: Should the module name match the artifact name?  (CORE_X12DocumentSubmission_10)

File: Product/Production/Gateway/X12DocumentSubmission_10/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/entity/EntityX12DocumentSubmissionUnSecured.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import gov.hhs.fha.nhinc.common.nhinccommonentity.RespondingGatewayCrossGatewayBatchSubmissionRequestType;
+import gov.hhs.fha.nhinc.common.nhinccommonentity.RespondingGatewayCrossGatewayBatchSubmissionResponseMessageRequestType;
+import gov.hhs.fha.nhinc.common.nhinccommonentity.RespondingGatewayCrossGatewayRealTimeRequestType;
+import gov.hhs.fha.nhinc.common.nhinccommonentity.RespondingGatewayCrossGatewayRealTimeResponseType;
+
+/**
+ *
+ * @author svalluripalli
+ */
+public class EntityX12DocumentSubmissionUnSecured implements gov.hhs.fha.nhinc.nhincentitycore.EntityCORETransactionPortType {
```
Comment (christophermay07) [inline Product/Production/Gateway/X12DocumentSubmission_10/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/entity/EntityX12DocumentSubmissionUnSecured.java]: Consider renaming to "Unsecured" (no capital 'S') to stay consistent with other CONNECT Unsecured classes.

File: Product/Production/Deploy/ear/pom.xml [hunk 3]
```diff
@@ -77,6 +77,13 @@
             <version>${project.parent.version}</version>
             <type>pom</type>
         </dependency>
+		<!-- Core X12 -->
```
Comment (christophermay07) [inline Product/Production/Deploy/ear/pom.xml]: Changes under the Deploy folder should be saved for CONN-1285

File: Product/Production/Gateway/pom.xml (hunk 1 above)
Comment (vsaibabu) [inline Product/Production/Gateway/pom.xml]: Fixed this to match the module name.

File: Product/Production/Gateway/X12DocumentSubmission_10/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/entity/EntityX12DocumentSubmissionUnSecured.java (hunk 2 above)
Comment (vsaibabu) [inline Product/Production/Gateway/X12DocumentSubmission_10/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/_10/entity/EntityX12DocumentSubmissionUnSecured.java]: Remove 'S' and updated class with Unsecured

**Pull request closed without merging**